streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobScheduler.scala

```diff
@@ -163,15 +163,35 @@ class JobScheduler(val ssc: StreamingContext) extends Logging {
       listenerBus.post(StreamingListenerBatchStarted(jobSet.toBatchInfo))
     }
     listenerBus.post(StreamingListenerOutputOperationStarted(
-      OutputOperationInfo(job.time, job.outputOpId, job.callSite, Some(startTime), None)))
+      OutputOperationInfo(
+        job.time,
+        job.outputOpId,
+        job.callSite.shortForm,
```

Comment:
And `StageInfo` gets its `name` and `details` just from `Stage.name` and `Stage.details`.